.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
+
+on:
```

Comment:
This was not originally intended for this PR, however it was easy to add while i was working on the C++ so I did.  I'm happy to remove it.